activesupport/CHANGELOG.md

```diff
@@ -1,22 +1,30 @@
-*   Override `inspect` methods in `CombinedConfiguration` and `EnvConfiguration` to prevent
-    sensitive ENV variables and credentials from being exposed in console or logs.
-    This matches the existing security behavior of `EncryptedConfiguration`.
+*   Override `inspect` methods in `CombinedConfiguration`, `EnvConfiguration`, and
+    `EncryptedConfiguration` to show key names while hiding sensitive values. This prevents
+    accidental exposure of secrets in console or logs while still providing useful debugging
+    information about what configuration keys are available.
 
     Before:
 
     ```ruby
+    Rails.app.creds
     #<ActiveSupport::CombinedConfiguration:0x0000000122b45870
      @configurations=[
        #<ActiveSupport::EnvConfiguration:0x0000000122b465b8
-        @envs={"SECRET" => "SECRET_MSG", ... }>,
+        @envs={"SOME_SECRET" =>  "secret_value", ...}>,
        #<ActiveSupport::EncryptedConfiguration:0x00000000007af0>]>
     ```
 
     After:
 
     ```ruby
     Rails.app.creds
-    => #<ActiveSupport::CombinedConfiguration:0x0000000122b45870>
+    #<ActiveSupport::CombinedConfiguration:0x00000000006490 keys=["RAILS_ENV", "SOME_SECRET", ..., :secret_key_base, :stripe_webhook_secret]>
```

Comment:
We should be converting all these keys to symbols, since that's how you're going to access them.